src/qt/bitcoingui.cpp

```diff
@@ -672,13 +678,13 @@ void BitcoinGUI::setClientModel(ClientModel *_clientModel)
         }
 #endif // ENABLE_WALLET
         unitDisplayControl->setOptionsModel(_clientModel->getOptionsModel());
-        
+
```

Comment:
I don't really care too much, but we normally avoid making formatting changes in the code that we don't touch. This is in order to avoid merge conflicts later when backporting bitcoin code